source/agora/consensus/data/Block.d

```diff
@@ -468,7 +470,7 @@ unittest
 *******************************************************************************/
 
 public Block makeNewBlock (Transactions)(const ref Block prev_block,
-    Transactions txs, Enrollment[] enrollments) @safe nothrow
+    Transactions txs, Enrollment[] enrollments = null) @safe nothrow
```

Comment:
I don't think it's a good idea to use default parameters for functions which are used in CoinNet. The enrollments should always be explicitly passed, even if they're empty. Otherwise we risk the situation of forgetting to include them when creating a block.